MIDAS/src/sensor_data.h

```diff
@@ -63,8 +63,19 @@ struct Barometer {
     float pressure = 0;
 };
 
+// Detects if a charge or 
 struct Continuity {
-    bool is_continuous = false;
+    Continuity() = default;
```

Comment:
remove the default constructor, just initialize it with the {} in the update function